backend/app/models.py

```diff
@@ -37,6 +46,31 @@ class FlathubUser(Base):
 
     TABLES_FOR_DELETE = []
 
+    def connected_accounts(self, db) -> list["ConnectedAccount"]:
+        result = []
+        for table in ConnectedAccountTables:
+            if account := table.by_user(db, self):
+                result.append(account)
+        return result
+
+    def get_connected_account(self, db, provider: str) -> Optional["ConnectedAccount"]:
```

Comment:
Can we type the provider? Maybe with a union? Not sure if that works in python.

Reply:
Changed it to use a string enum.